inference-engine/src/vpu/graph_transformer/src/utils/runtime_graph.cpp

```diff
@@ -241,8 +241,13 @@ std::map<std::string, std::string> extractMeta(const StageMetaInfo& stageMeta) {
     serializationInfo[ExecGraphInfoSerialization::OUTPUT_LAYOUTS] = layoutStream.str();
 
     std::string outPrecisionsStr;
+    auto runtimePrecision = Precision(Precision::I32);
     ind = 0;
     for (auto &outPrecision : stageMeta.outPrecisions) {
+        // if we have any output precision not equal I32 -> we assume runtimePrecision is FP16
+        if (outPrecision != Precision::I32) {
+            runtimePrecision = Precision(Precision::FP16);
```

Comment:
Do we really need to use an explicit constructor? 
Does `runtimePrecision = Precision::FP16;` work?